mirdata/rwc_classical.py

```diff
@@ -0,0 +1,237 @@
+    filename='AIST.RWC-MDB-C-2001.BEAT.zip',
+    url='https://staff.aist.go.jp/m.goto/RWC-MDB/AIST-Annotation/AIST.RWC-MDB-C-2001.BEAT.zip',
+    checksum='e8ee05854833cbf5eb7280663f71c29b')
+RWC_CLASSICAL_ANNOT_REMOTE_2 =  utils.RemoteFileMetadata(
+    filename='AIST.RWC-MDB-C-2001.CHORUS.zip',
+    url='https://staff.aist.go.jp/m.goto/RWC-MDB/AIST-Annotation/AIST.RWC-MDB-C-2001.CHORUS.zip',
+    checksum='f77bd527510376f59f5a2eed8fd7feb3')
+
+
+RWCClassicalTrack = namedtuple(
+    'RWCClassicalTrack',
+    ['track_id',
+     'audio_path',
+     'sections',
+     'beats',
+     'duration_sec',
+     'title',
+     'artist',
+     'variation',
+     'instruments']
+)
+
+
+def download(data_home=None, clobber=False):
+    save_path = utils.get_save_path(data_home)
+    dataset_path = os.path.join(save_path, RWC_CLASSICAL_DIR, 'annotations')
+
+    if clobber:
+        utils.clobber_all(RWC_CLASSICAL_ANNOT_REMOTE_1,
+                          dataset_path,
+                          data_home)
+        utils.clobber_all(RWC_CLASSICAL_ANNOT_REMOTE_2,
+                          dataset_path,
+                          data_home)
+
+    if utils.check_validated(dataset_path):
+        print("""
+                The {} dataset has already been downloaded and validated.
+                Skipping download of dataset. If you feel this is a mistake please
+                rerun and set clobber to true
+                """.format(RWC_CLASSICAL_DIR))
+        return
+
+    download_path_1 = utils.download_from_remote(
+        RWC_CLASSICAL_ANNOT_REMOTE_1, data_home=data_home, clobber=clobber)
+    download_path_2 = utils.download_from_remote(
+        RWC_CLASSICAL_ANNOT_REMOTE_2, data_home=data_home, clobber=clobber)
+
+    if not os.path.exists(dataset_path):
+        os.makedirs(dataset_path)
+
+    utils.unzip(download_path_1, dataset_path, cleanup=True)
+    utils.unzip(download_path_2, dataset_path, cleanup=True)
+
+    missing_files, invalid_checksums = validate(dataset_path, data_home)
+    if missing_files or invalid_checksums:
+        print("""
+            Unfortunately the audio files of the RWC-Classical dataset are not available
+            for download. If you have the RWC-Classical dataset, place the contents into a
+            folder called RWC-Classical with the following structure:
+                > RWC-Classical/
+                    > annotations/
+                    > audio/
+            and copy the RWC-Classical folder to {}
+        """.format(save_path))
+
+    # metadata
+    utils.download_from_remote(
+            RWC_CLASSICAL_METADATA, data_home=data_home, clobber=clobber)
+
+
+def validate(dataset_path, data_home=None):
+    missing_files, invalid_checksums = utils.validator(RWC_CLASSICAL_INDEX, data_home, dataset_path)
+    return missing_files, invalid_checksums
+
+
+def track_ids():
+    return list(RWC_CLASSICAL_INDEX.keys())
+
+
+def load(data_home=None):
+    validate(data_home)
```

Comment:
validate takes two arguments not one. Copy paste the following lines from download into here (at some point we're going to have to make that its own function...)
```
    save_path = utils.get_save_path(data_home)
    dataset_path = os.path.join(save_path, RWC_CLASSICAL_DIR, 'annotations')
```